templates/monorepo/packages/react/package.json

```
@@ -0,0 +1,36 @@
+{
+  "name": "@mono/react",
+  "version": "0.0.0",
```

```suggestion
  "version": "0.0.0",
  "license": "MIT",
```

MIT license missing. We probably want to include a `LICENSE` file in each package as well. NPM will automatically include any `LICENSE` file when publishing (similar to `README.md`)

---

Lerna copies root license into each package before publishing

---

Oh, does it do that for `package.json.license` too? Fancy